data/README.md

```diff
@@ -28,7 +28,9 @@
 | delta_encoding_optional_column.parquet       | optional INT64 and STRING columns with delta encoding. See [delta_encoding_optional_column.md](delta_encoding_optional_column.md) for details.                  |
 | nested_structs.rust.parquet                  | Used to test that the Rust Arrow reader can lookup the correct field from a nested struct. See [ARROW-11452](https://issues.apache.org/jira/browse/ARROW-11452) |
 | data_index_bloom_encoding_stats.parquet | optional STRING column. Contains optional metadata: bloom filters, column index, offset index and encoding stats.                                               |
-|null_list.parquet                             | an empty list. Generated from this json `{"emptylist":[]}` and for the purposes of testing correct read/write behaviour of this base case.                 |
+| null_list.parquet                       | an empty list. Generated from this json `{"emptylist":[]}` and for the purposes of testing correct read/write behaviour of this base case.                      |
+| alltypes_tiny_pages.parquet             | small page sizes with dictionary encoding with page index                                                                                                       |
+| alltypes_tiny_pages_plain.parquet       | small page sizes without dictionary encoding with page index                                                                                                    |
```

Comment:
Maybe we can also mention the file's lineage (that it came from impala, with link) in this comment